xbmc/guilib/GUIMoverControl.dox

```diff
@@ -37,10 +43,42 @@ important, as xml tags are case-sensitive.
 |----------------:|:--------------------------------------------------------------|
 | texturefocus    | Specifies the image file which should be displayed when the mover has focus. [See here for additional information about textures](http://kodi.wiki/view/Texture_Attributes).
 | texturenofocus  | Specifies the image file which should be displayed when the mover does not have focus.
+| movingspeed     | Specifies the allowed directional movements and their configuration. The <c>`<movingspeed>`</c> must contains as many <c>`<eventconfig>`</c> tags as required.
```

Comment:
Specifies the allowed directional movements and respective configurations. The <c>`<movingspeed>`</c> can contain as many <c>`<eventconfig>`</c> tags as required.

Reply:
thank you this and all other changes applied